src/equations/acoustic_perturbation_2d.jl

```diff
@@ -127,12 +128,13 @@ A smooth initial condition used for convergence tests in combination with
 """
 function initial_condition_convergence_test(x, t,
                                             equations::AcousticPerturbationEquations2D)
-    c = 2.0
-    A = 0.2
-    L = 2.0
-    f = 2.0 / L
-    a = 1.0
-    omega = 2 * pi * f
+    RealT = eltype(x)
+    a = 1
+    c = 2
+    L = 2
+    f = 2 / L
```

Comment:
```suggestion
    L = convert(RealT, 2) # since we divide by L below
    f = 2 / L
```

Reply:
Interesting. I have never thought about this. Why integer division is designed like this?

Reply:
It's not (truncated) integer division (which would be `div`). Yielding floating point values is what you would like to have in scientific computing. Thus, you need to choose a default floating point type - `Float64` in Julia (also used for `sqrt` etc.).

Reply:
Thanks for the detailed reply!

Reply:
Please consider the suggestion I made above

Reply:
Sure! I am currently working on the next file. I will make a commit together.